homeassistant/components/config/device_registry.py

```diff
@@ -51,6 +33,11 @@ def _async_clear_list_device_cache(event: Event) -> None:
         cached_list_devices = None
 
     @callback
+    @websocket_api.websocket_command(
+        {
+            vol.Required("type"): "config/device_registry/list",
+        }
+    )
     def websocket_list_devices(hass, connection, msg):
```

Comment:
Could you add type hints here at the same time?
```suggestion
    def websocket_list_devices(
        hass: HomeAssistant,
        connection: websocket_api.ActiveConnection,
        msg: dict[str, Any],
    ) -> None:
```